src/main/java/com/hotels/bdp/circustrain/bigquery/partition/PartitionColumnFormatter.java

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (C) 2018 Expedia Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hotels.bdp.circustrain.bigquery.partition;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.hadoop.hive.metastore.api.FieldSchema;
+
+import com.google.api.client.util.Joiner;
+
+public class PartitionColumnFormatter {
+
+  private static Joiner joiner = Joiner.on(',');
+
+  public static String getFormattedListOfColumns(List<FieldSchema> columns) {
```

Comment:
This isn't a getter and it's not returning a list so the method name is a bit misleading. Perhaps just `formatColumns` would be better?